tests/Microsoft.Bot.Builder.Tests/MemoryConversations.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
```

Comment:
I don't understand why this file is included.

Reply:
It's used in my tests.

Reply:
Tests...look at my tests.